Gemfile

```
@@ -31,7 +32,8 @@ group :development do
 end
 
 group :test do
-  gem 'rspec-rails'
+  gem 'capybara'
   gem 'shoulda-matchers'
   gem 'simplecov'
+  gem 'webdrivers', '~> 4.0', require: false
```

```suggestion
  gem 'webdrivers'
```

Couple of things:
- if we omit `require: false`, it actually gets rid of the need to `require 'webdrivers'` explicitly in the `rails_helper` file (this might be another thing I'd disagree with the guide about).
- we also don't need to pin this gem to any particular version (the `'~> 4.0'` part). This actually restricts us to a version of webdrivers that is one major version behind - the latest version is `5.2.0` (I'm guessing the guide was written before the latest was released)

After you remove the pin, be sure to run the command `bundle update webdrivers`, which should update the version of webdrivers within the lockfile.